### ISSUE TITLE
identd

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#595]
*Issue migrated from trac ticket [#314](http://colloquy.info/project/ticket/314)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** Normal | **version:** 2.0 (Mac) | **status:** closed | **resolution:** Invalid
___
#### 2005-09-12 16:49:31: condond@… created the issue

how do u install identd for calloquay
___
#### 2005-09-12 20:09:23: @kballard changed status from *new* to *closed*

___
#### 2005-09-12 20:09:23: @kballard changed resolution from ** to *invalid*

___
#### 2005-09-12 20:09:23: @kballard commented

This is not something that should be a ticket. This is something that should be asked on the [irc://irc.freenode.net/#colloquy IRC channel].

Basically, you can install identd however you wish - it has absolutely nothing to do with Colloquy (it affects all IRC clients equally). Also note that identd won't work if you're behind NAT or a restrictive firewall.
